content/en/docs/setup/production-enviroments.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Production Environments
```

Comment:
I see the point of this page, but I don't like the title. Can we change it to "Specific Environments" or something similar?

Reply:
+1 for `Specific Environments` til we get better